modern/tests/test_installed/test_installed.py

```diff
@@ -9,7 +9,7 @@
     "perl": {"16.04": "5.22.1"},
     "wget": {"16.04": "1.17.1"},
     "curl": {"16.04": "7.47.0"},
-    "git": {"16.04": "2.7.4"},
+    "git": {"16.04": "2.34.1"},
```

Comment:
Probably new versions will arrive to `ppa:git-core/ppa`, so we need to install a fixed one.

Can we use in the docker command something like...
```
apt-get install -y --no-install-recommends --no-install-suggests \
   ...
   git=2.34.1
   ...
```

Reply:
Not needed. There is not option when a version become available on PPA. It's like Highlader, only one can exist. https://launchpad.net/~git-core/+archive/ubuntu/ppa?field.series_filter=xenial When a new version become available, 2.34.1 will be removed from there and will break the installation. It's a normal process.  The only way to keep more than one version, is uploading deb packages to Artfactory, but it's something that not inclined to manage.

Reply:
Then we should change the test for something like `>=2.34.1`. The CI has to work. Being `git` I'm pretty sure it will keep working, so probably there is no need to run Git commands... but just in case I would also check major version `<3`